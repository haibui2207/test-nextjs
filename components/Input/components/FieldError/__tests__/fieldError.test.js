import React from 'react';
import { shallow } from 'enzyme';

import FieldError from '../FieldError';

describe('Testing Input component > FieldError', () => {
  it('should render match snapshot', () => {
    const wrapper = shallow(<FieldError error="Some error" />);
    expect(wrapper).toMatchSnapshot();
  });
});
