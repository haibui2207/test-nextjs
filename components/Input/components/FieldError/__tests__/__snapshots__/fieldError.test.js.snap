// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Input component > FieldError should render match snapshot 1`] = `ShallowWrapper {}`;
